atomic_reactor/core.py

```diff
@@ -230,12 +278,14 @@ def __init__(self, base_url=None, timeout=120, **kwargs):
         if hasattr(docker, 'AutoVersionClient'):
             client_kwargs['version'] = 'auto'
 
-        try:
-            # docker-py 2.x
-            self.d = docker.APIClient(**client_kwargs)
-        except AttributeError:
-            # docker-py 1.x
-            self.d = docker.Client(**client_kwargs)
+        self.retry_times = retry_times
+        if not str(retry_times).isdigit() or retry_times == 0:
```

Comment:
well it can be passed as anything, so this just makes sure that wouldn't break retries, if for some reason somebody set that constant incorrectly